assets/javascripts/discourse/initializers/event-edits.js.es6

```diff
@@ -296,15 +296,15 @@ export default {
       }
 
       api.modifyClass('controller:topic', {
-        @observes('model.id')
+        @observes('model.username')
```

Comment:
@fzngagan This should not be changed. Maybe this one was caught as part of a mass id to username change? This is the topic id though. There is no topic.username.